Scripts/CI/README_Metadata_StyleCheck/metadata_style_checker.py

```diff
@@ -0,0 +1,338 @@
+import os
+import re
+import json
+import typing
+import argparse
+import glob
+from pathlib import Path
+
+# region Global sets
+# A set of category folder names in current sample viewer.
+categories = {
```

Comment:
Same here, if we could read this in from a file, it would save having to update the list in three places